stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -1303,6 +1303,50 @@ function UndefVarError_hint(io::IO, ex::UndefVarError)
     elseif var === :quit
         print(io, "\nsuggestion: To exit Julia, use Ctrl-D, or type exit() and press enter.")
     end
+    if isdefined(ex, :scope)
+        scope = ex.scope
+        if scope isa Module
+            bnd = ccall(:jl_get_module_binding, Any, (Any, Any, Cint), scope, var, true)::Core.Binding
+            if isdefined(bnd, :owner)
+                owner = bnd.owner
+                if owner === bnd
+                    print(io, "\nsuggestion: add an appropriate import or assignment. This global was declared by not assigned.")
```

Comment:
```suggestion
                    print(io, "\nSuggestion: add an appropriate import or assignment. This global was declared but not assigned.")
```

Reply:
Just noting that if "suggestion" should be capitalized, there are also a few additions in the docs that should be updated. 

And there are a few occurrences right above this that have the uncapitalized version, making for inconsistent usage if these are updated, though maybe it doesn't really fit in the scope of this PR to fix that.